packages/features/portfolio/src/containers/portfolio-overview/portfolio-overview.tsx

```diff
@@ -92,7 +92,7 @@ const PortfolioOverview = () => {
               }
             />
           )}
-          {vaultHoldingsData.length > 0 && (
+          {vaultHoldingsData[0]?.vault.symbol && (
```

Comment:
Is the array always guaranteed to have an entry? Is it the "total row"? Maybe we should make a view method on the PortfolioModel that returns a boolean if the account has vault holdings. Won't this same issue happen if the account does not have any portfolio holdings? Maybe we should do the same there.

Reply:
I updated the check to when we're setting the totalVaultHoldings. Lmk what you think